src/main/java/desktop/functionalActions/JavascriptActions.java

```diff
@@ -6,13 +6,14 @@
 import static webdriver.DriverManager.getDriver;
 
 public class JavascriptActions {
-    private static final JavascriptExecutor js = (JavascriptExecutor) getDriver();
 
     public static void elementHighlighter(WebElement element) {
+        JavascriptExecutor js = (JavascriptExecutor) getDriver();
```

Comment:
Avoid using redundant variable. If you wish to incapsulate cast, make method like
asJS() or withJS() or usingJsExecutor()

return (JavascriptExecutor) getDriver();